Directory.Build.targets

```diff
@@ -4,20 +4,20 @@
     <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Update="NJsonSchema" Version="10.6.6" />
```

Comment:
10.6.7 now

Reply:
Yup, I have to drop the 3/5 targets for the preview feed, but want to get this one out with 1.2.2 first